cpp/vm/evmzero/interpreter_test.cc

```diff
@@ -4853,6 +4853,29 @@ TEST(InterpreterTest, SELFDESTRUCT_StaticCallViolation) {
   });
 }
 
+///////////////////////////////////////////////////////////
+// CREATE
+
+TEST(InterpreterTest, CREATE_InsuffiecientBalanceIsHandledCorrectly) {
```

Comment:
There should be another test case testing that in a case where there is not enough balance the memory has grown to the size of the input data. Before this change, this used to be not the case.

Reply:
Done.